python/device/device.py

```diff
@@ -712,27 +764,31 @@ def dispatch_incoming_message_thread(self, worker_thread_info):
                 time.sleep(1)
                 continue
 
-            msg = self.client.receive_message(timeout=1)
+            msg = self.client.receive_message(timeout=30)
 
             if msg:
                 obj = json.loads(msg.data.decode())
                 thief = obj.get("thief")
 
                 if thief and thief["pairingId"] == self.pairing_id:
+                    # We only inspect messages that have `thief/pairingId` set to our `pairingId` value.
                     cmd = thief["cmd"]
                     if cmd == "pingbackResponse":
+                        # If this is a pingback response, we put it into `incoming_pingback_response_queue`
```

Comment:
What is pingback?

Reply:
This is the word I've been using for the service app acknowledging something.  telemetry and reported property changes both generate this ack.  I don't think it's a great name but I'm not sure what would be better.  Do you have any suggestions?

Reply:
ServiceAck?